gradle.properties

```diff
@@ -128,7 +128,7 @@ commonsCollectionsVersion=3.2.2
 commonsCollections4Version=4.4
 commonsCodecVersion=1.16.0
 # sync with version Tika ships
-commonsCompressVersion=1.24.0
+commonsCompressVersion=1.26.0
```

Comment:
We can probably remove this comment about Tika. This will be at least the second time we've bumped this without touching Tika.